templates/word-daily-level/src/addons/daily/daily.js

```diff
@@ -1,21 +1,52 @@
 import * as playpass from "playpass";
 import { getCurrentDay, getDay } from "./timer";
 
+/**
+ * A daily state manager
+ */
 export class Daily {
     constructor (firstDate) {
         this.day = getCurrentDay() - getDay(firstDate);
     }
 
-    /** Loads an object from storage, returning null if there was no object previously saved today. */
-    async loadObject () {
-        const state = await playpass.storage.get("daily");
-        const newState = {
-            words: [""],
-            marks: [],
+    /** 
+     * Defines the base object for user state
+     * 
+     * Override with your own implementation to customize what is persisted
+     * @abstract
+     */
+    data() {
+        return {};
+    }
```

Comment:
We should probably move global user state somewhere else and keep `Daily` just for daily state.